.lintstagedrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
nit/optional: can we nest this file somewhere to reduce the number of files in the root folder?

Reply:
we can nest this file in package.json

Reply:
Done